env-cloud.json

```diff
@@ -0,0 +1,18 @@
+{
+  "MQ_ENDPOINTS": [{
+      "HOST": "test-a95a.qm.us-south.mq.appdomain.cloud",
+      "PORT": "32055",
+      "CHANNEL": "CLOUD.APP.SVRCONN",
+      "QMGR": "test",
+      "APP_USER": "credential2",
+      "APP_PASSWORD": "gOojnDA9y14IXhIH7gnKJA5AeTS_PFbz4FKda1KLuwSS",
```

Comment:
Remember this is a public repository and all credentials are visible for ever, all it takes is to find the right commit. You should ensure that the credentials you have shared here are no-longer valid. It isn't sufficient to remove them as they can be refound.

Reply:
The instance on the Cloud has been deleted.